mm2src/mm2_main/src/docker_tests.rs

```diff
@@ -430,6 +430,7 @@ mod docker_tests {
     }
 
     #[test]
+    #[ignore]
```

Comment:
The same this test

Reply:
Was ignored by me, will recheck the reason and likely turn the test back on.

Reply:
I have remembered 🙂 Ignored because of this: https://github.com/KomodoPlatform/atomicDEX-API/issues/1473. As the issue is fixed, I removed `#[ignore]`.